### PR TITLE
added LDAP group prune doc

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Follow-up to https://github.com/openshift/openshift-docs/pull/1066

Dependent on https://github.com/openshift/origin/pull/5709

/cc @deads2k @adellape

### CODE REVIEW

**Comment by adellape** [comment]
FYI, I moved this topic up into the Install & Config guide yesterday:

https://docs.openshift.org/latest/install_config/syncing_groups_with_ldap.html

Which would affect the `build_cfg` in this PR.

**Comment by stevekuznetsov** [comment]
@adellape Updated locations.

**Comment by stevekuznetsov** [comment]
@deads2k Let me know how much more doc we need for pruning.

**Comment by stevekuznetsov** [comment]
@deads2k PTAL

File: install_config/syncing_groups_with_ldap.adoc [hunk 1]
```diff
@@ -186,6 +186,14 @@ $ openshift ex sync-groups group_unique_idenitifer   \
 $ openshift ex sync-groups --type=openshift --whitelist=whitelist.txt --sync-config=config.yaml --confirm
 ----
 
+== Running a Group Pruning Job
+An administrator can also choose to remove groups from OpenShift records if the records on the LDAP
+server that created them are no longer present. The prune job will accept the same sync
+configuration file and white- or black-lists as used for the sync job:
+----
+$ openshift ex prune-groups --sync-config=config.yaml --confirm
```
Comment (deads2k) [inline install_config/syncing_groups_with_ldap.adoc]: Tell me what this command is doing.

**Comment by deads2k** [comment]
one comment, lgtm to me otherwise.

The code is queue: https://github.com/openshift/origin/pull/5709

**Comment by stevekuznetsov** [comment]
@deads2k addressed comment
@adellape Origin code merged, and I think this pull is ready now.

**Comment by stevekuznetsov** [comment]
folded in upgrade changes as per https://github.com/openshift/openshift-docs/pull/1363. no further action needed for this PR I believe

**Comment by adellape** [comment]
@stevekuznetsov This PR title would more accurately be more like "added LDAP group prune doc" now, yea?

**Comment by stevekuznetsov** [comment]
@adellape Yes, updated.

**Comment by adellape** [comment]
Thanks, LGTM.

**Pull request merged**